.licenserc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "**/*.py": "# Copyright "
```

Comment:
```suggestion
  "**/*.py": [
  "# Copyright ",
  "#",
  "# Licensed under the Apache License, Version 2.0 (the \"License\");"
  ],
  "**/*.{h,cpp,hpp}": [
  "// Copyright", 
  "//",
  "// Licensed under the Apache License, Version 2.0 (the \"License\");"
  ]
```

Reply:
Rationale: 
We have C++ as well as Python in the project
We should not allow to include non-Apache Licenses in our core (we can add 3rd party src folder and exclude it from this requirement to allow for compatible licenses such as MIT, BSD 3, Mozilla later).

Reply:
I've tested it using `npx @kt3k/license-checker`
with config
```json
{
    "**/*.py": [
        "# Copyright ",
        "#",
        "# Licensed under the Apache License, Version 2.0 (the \"License\");"
    ],
    "**/*.{h,cpp,hpp}": [
        "// Copyright ",
        "//",
        "// Licensed under the Apache License, Version 2.0 (the \"License\");"
    ]
}
```

Reply:
Great! How about you either apply + modify my suggestion or submit your patch (tested) with my co-authorship?

Reply:
Up to @john-science

Reply:
Yeah, that change is solid. Taken.

I was only looking at the Python code, my bad.